cdap-docs/reference-manual/source/http-restful-api/metadata.rst

```diff
@@ -1207,6 +1208,336 @@ Rolling up the above using ``rollup=workflow`` would group the programs together
 
 .. highlight:: console
 
+.. _http-restful-api-metadata-fieldlevellineage:
+
+Field Level Lineage
+===================
+
+Fields associated with the Dataset
+----------------------------------
+
+Get the fields which were associated with the dataset in a given time range::
```

Comment:
Gets the fields that were associated with the dataset for the specified range of time.

Reply:
done.